src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -2521,13 +2521,15 @@ export class PatientViewPageStore {
                 // - custom driver annotations are part of the incoming datum
                 // - cbio counts, cosmic and custom driver annnotations are
                 //   not used for driver evaluation
-                return evaluateMutationPutativeDriverInfo(
+                return evaluatePutativeDriverInfoWithHotspots(
                     mutation,
                     oncoKbDatum,
-                    true,
-                    isHotspotDriver,
                     false,
-                    undefined
+                    undefined,
+                    {
+                        hotspotAnnotationsActive: true,
```

Comment:
@BasLee  did you mean to hardcode true here?

Reply:
yes, in the original code `hotspotAnnotationsActive` is the third parameter of `evaluateMutationPutativeDriverInfo`